### PR TITLE
Fix documentation errors [UG] assigned to Russell

### LINKED ISSUE #272: [PE-D][Tester D] Unexpected behaviour in tag and untag feature
In the UG, it was stated that issuing a tag command like `tag m/CS2107` without providing an index is only supposed to add the module tag to the user's own profile. 

However, executing `tag m/CS2107` results in the tag being added to other contacts within the address book, as illustrated below.

_Before running command_

_After running command_

The above behaviour also applies to untag commands. 

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: huixuant/ped#1

### LINKED ISSUE #289: [PE-D][Tester B] Screenshot does not Correspond to Command in UG
Under the untag command, the screenshot shows the tag command instead. 

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: cmang12/ped#5

### LINKED ISSUE #291: [PE-D][Tester B] Typo in UG
There is a typo in the word Module in fifth bullet point.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: cmang12/ped#6

### LINKED ISSUE #301: [PE-D][Tester C] Too many similar diagrams in the UG
Issue: Took me a very long time to tell what the difference is/the focus on the diagrams because these diagrams are too similar with the only difference being the command result. 

Suggestion: Perhaps highlight to the user what is to be noted.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: cyiting/ped#10

### LINKED ISSUE #303: [PE-D][Tester D] Duplicate formats listed for tag and untag in UG
In the section "Add a label to a contact" in the UG, the third and fifth formats are repeated, with a typo in the fifth point. 

The above applies to "Remove a label from a contact" as well.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: huixuant/ped#7

### LINKED ISSUE #308: [PE-D][Tester A] Difference between `tag` and the `edit` feature
## Context
Currently, both the `tag` and `edit` feature allows the user to add modules to an existing user. 
However, there seems to be no difference between them. They seem to function in the same manner, which is that they add a module or group. As an end-user, this feature may seem redundant or confusing. 

Perhaps for `tag` a user might expect tagging a person as `important`, `bestie` or smtg instead? This tag would preferable provide some utility value to the user. E.g. maybe it acts somewhat like a "pin" function, or allows easy search by just searching the tag. 

Refer to below steps for an example. 

## Step to reproduce 
1. Run `view` to show user info
2. Run `tag 0 m/[some valid module]`

## Behaviour
The module appears under in the user profile.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: ShanHng/ped#8

### LINKED ISSUE #311: [PE-D][Tester A] Ambiguity in actual behavior of Person List with reference to the UG
It seems like Person List only display module tags that are **shared** between the user and the persons on the Person List. Issue #3 and Issue #9 seem to be due to this undocumented behavior. 

Might be good to mention this, if this is the intended behaviour of your application, under the Person List segment of your UG. 

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: ShanHng/ped#10

### LINKED ISSUE #312: [PE-D][Tester A] Supplementing the purpose of the `list` command in the UG
As a reader, I am unsure as to why I will need this command at first read, as I am unaware of the
list-filtering features that are available below. It would be good to bring up why this `list` command is needed here.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: ShanHng/ped#11

### LINKED ISSUE #313: [PE-D][Tester A] Naming of `Explanation` segment under UG as `Example of Usage` instead
I love that you guys provide examples/scenarios for your functions. However, I feel like 'Example of Usage' or something that indicates the presence of an example use case might be a better name for the `Explanation` segment. 

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: ShanHng/ped#12

### LINKED ISSUE #314: [PE-D][Tester A] Purpose of different panels might be unclear to novice users
Love the `meet` feature, it's rly useful. However, both the Meet Up Reccommendation Panel and the Schedule Meet Up Panel display almost similar information. Without labels, these panels might be confusing for a new/novice user. 

-------------
Labels: `type.FeatureFlaw` `severity.VeryLow`
original: ShanHng/ped#13

### LINKED ISSUE #315: [PE-D][Tester A] `organize` does not work as advertised
The `organize` feature is not recognized at all. I inferred this information from the Command Response box (refer to the screenshot).

## Step to reproduce 
1. Run `organize 1`

## Screenshot

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: ShanHng/ped#14

### LINKED ISSUE #326: Include Sample Command Description for UG

### PR DESCRIPTION
**Pull request by russelltankaimin**
## Things to check:
* **Structure**
* **Flow**
* **GRAMMAR (TENSES are CONSISTENT THROUGHOUT)**
* **READABILITY**

## [UG Click here](https://russelltankaimin.github.io/tp/UserGuide.html)

- [x] Closes #314 
- [x] Closes #311 
- [x] Closes #326 
- [x] Closes #312 
- [x] Closes #313 
- [x] Closes #301 
- [x] Closes #303 
- [x] Closes #315  
- [x] Closes #308 
- [x] Closes #291 
- [x] Closes #289 
- [x] Closes #272

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -764,22 +972,34 @@ We will now teach you how to effectively use the `sort` command.
 
 </div>
 
+`sort` is one of the most complicated commands to get your head around. Perhaps a scenario can help you understand better!
+
+##### Scenario:
+
+Suppose you want to arrange contacts such from those that take the most number of common modules as you to the least.
```
Comment (mynameizzhafeez) [inline docs/UserGuide.md:979]: Sentence does not make sense.

"Suppose you want to arrange your contacts based on the number of common modules they have with you, from those who have the most in common to those who have the least."

File: docs/UserGuide.md [hunk 2]
```diff
@@ -764,22 +972,34 @@ We will now teach you how to effectively use the `sort` command.
...
 
 <br>
 
-##### Explanation:
+##### Extra Explanation:
 
 The command `sort m/d n/a` would mean that `EduMate` would first sort contacts by decreasing number of common modules with you, and tie-break by increasing alphabetical order.
 
-Hence, Alex Quinn is ranked higher than Astrid Holland as Alex's name is lexicographically smaller than Astrid even though they share the same common modules as you.
+Hence, Alex Quinn is ranked higher than Amirul as Alex's name is lexicographically smaller than Amirul even though they share the same common modules as you.
```
Comment (mynameizzhafeez) [inline docs/UserGuide.md:997]: "they both share 2 modules with you"

File: docs/UserGuide.md [hunk 3]
```diff
@@ -851,18 +1071,26 @@ The `meet` command in EduMate is used to suggest suitable meet-up locations and
...
 </div>
 
+##### Scenario:
+
+After sorting your contacts, you decided to ask Tae Seong, Alex, Amirul and Kevin from your CCA to study for CS2100. 
+
+Problem is, you do not know when all of them are free! Luckily, you have `EduMate` to find out for you!
+
+You can execute the `study` variant of the `meet` command
+whilst referring to their index on the Person List and find out what are the most optimal places and the best possible time to meet and study for the final.
```
Comment (mynameizzhafeez) [inline docs/UserGuide.md:1081]: Sentence is too long. Can be split by the first "and".

"You can execute the `study` variant of the `meet` command
whilst referring to their index on the Person List"

"From there, EduMate will recommend..."

File: docs/UserGuide.md [hunk 4]
```diff
@@ -875,7 +1103,20 @@ whilst referring to their index on the Person List and find out what are the mos
 
 ### Schedule Commands
 
-The `organise` command in EduMate is used to add meet ups with selected contacts, whether they are customised or recommended by the meet commands. It is useful for keeping track of your upcoming meetups in an organised and efficient way. In addition, EduMate also offers the `unorganise` command, which helps you manage and clean up scheduled meetups by removing them - making it easy to stay on top of your schedule.
+The `organise` command in EduMate is used to add meet ups with selected contacts, whether they are customised or recommended by the meet commands. It is useful for keeping track of your upcoming meetups in an organised and efficient way.
```
Comment (mynameizzhafeez) [inline docs/UserGuide.md:1106]: "The feature is useful"

File: docs/UserGuide.md [hunk 5]
```diff
@@ -875,7 +1103,20 @@ whilst referring to their index on the Person List and find out what are the mos
 
 ### Schedule Commands
 
-The `organise` command in EduMate is used to add meet ups with selected contacts, whether they are customised or recommended by the meet commands. It is useful for keeping track of your upcoming meetups in an organised and efficient way. In addition, EduMate also offers the `unorganise` command, which helps you manage and clean up scheduled meetups by removing them - making it easy to stay on top of your schedule.
+The `organise` command in EduMate is used to add meet ups with selected contacts, whether they are customised or recommended by the meet commands. It is useful for keeping track of your upcoming meetups in an organised and efficient way. 
+
+In addition, EduMate also offers the `unorganise` command, which helps you manage and clean up scheduled meetups by removing them - making it easy to stay on top of your schedule.
```
Comment (mynameizzhafeez) [inline docs/UserGuide.md:1108]: In addition + also mean the same thing, so can drop one.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -875,7 +1103,20 @@ whilst referring to their index on the Person List and find out what are the mos
...
+
+Any organised meet ups will be displayed at `EduMate`'s [Scheduled Meet Up Panel](#scheduled-meet-up-panel).
+
+<div markdown="span" class="alert alert-danger">
+
+:warning:**Common Mistake**
+
+The American spellings, `organize` and `unorganize` are not accepted by `EduMate`.
```
Comment (mynameizzhafeez) [inline docs/UserGuide.md:1116]: Good catch for the "special label" people.

File: docs/UserGuide.md [hunk 7]
```diff
@@ -921,32 +1170,52 @@ If you wish to remove a scheduled meetup from your [Scheduled Meet Up Panel](#sc
...
-<div markdown="block" class="alert alert-success">
 
-:bulb: **Example : Removing a meet up from the scheduled meet up list**<br>
-* `unorganise 1`: Removes the scheduled meet up with index 1 on the [Scheduled Meet Up Panel](#scheduled-meet-up-panel).
+##### Scenario:
 
-</div>
+Suppose Tae Seong has got Covid-19 and cannot make it to the meet up. Unfortunately, you have to cancel the meet up.
+
+The scheduled meet up has the index 1 on the [Scheduled Meet Up Panel](#scheduled-meet-up-panel).
```
Comment (mynameizzhafeez) [inline docs/UserGuide.md:1178]: Too many occurrences of "meet up" in quick succession. Perhaps spacing them out a bit.

"Suppose Tae Seong cannot attend the meet up due to contracting Covid-19, and you have to cancel the scheduled meet up."

This way, "meet up" are further apart.

**Review by mynameizzhafeez — COMMENTED** [review]
LGTM.

File: docs/UserGuide.md
[the remark this answers is not in the document]
Comment (russelltankaimin) [reply docs/UserGuide.md:1]: Yes. See pe-d code bug fix pr

[the remark this answers is not in the document]
Comment (russelltankaimin) [reply docs/UserGuide.md:1]: bottom of the application you mean

[the remark this answers is not in the document]
Comment (russelltankaimin) [reply docs/UserGuide.md:1]: Thanks Prof GPT...

File: docs/UserGuide.md (hunk 6 above)
Comment (russelltankaimin) [reply docs/UserGuide.md:1116]: Lets not go there.

File: docs/UserGuide.md
[the remark this answers is not in the document]
Comment (zichen-3974) [reply docs/UserGuide.md:1]: 5. Modules taken in current NUS semester
6. Groups

[the remark this answers is not in the document]
Comment (zichen-3974) [reply docs/UserGuide.md:1]: "It will provide feedback on command execution status(e.g success, invalid format)"

**Comment by kennycjy** [comment]
image

We should include the difference of tagging adds modules, while editing overwrites:
`tag 1 m/CS2100` adds CS2100 to person 1, while
`edit 1 m/CS2100` completely wipes the modules that person 1 has, and adds CS2100

**Comment by kennycjy** [comment]
image


We should standardise the steps:

> Step 1.

 instead of just 

> Step 1

image

File: docs/UserGuide.md
Comment (seanfirefox) [inline docs/UserGuide.md:1]: Should a picture of the location of the help and exit button be added here?
image

Comment (russelltankaimin) [reply docs/UserGuide.md:1]: not necessary.

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Suggestion to add in the numbers for the location (7 and 8 for help and exit button)
Might be good to add the link of the breakdown of EduMate's UI for exit as well
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Are these 2 hyperlinks meant to be linked to the same place?
image
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Side ToC exit does not work

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Side ToC exit does not work
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Suggestion to link this to the `java11` download page in Oracle instead
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Should this be labelled?
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Is it possible to rephrase number 5?
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Suggestion to add link to return to the top of the Commands.
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Typo?
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Do we need to wrap this in a tip?

image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: The example does not match the description of positive numbers. Consider changing to only numerical symbols is accepted to match the example and changing the example to `delete -1` for the description.
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Should not have spacing for website?
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Suggestion to place this in a warning box so that it is easier for the user to see.
image

Comment (russelltankaimin) [reply docs/UserGuide.md:1]: begin off yes

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Suggestion to put `view n/FULL_NAME` instead
image

Comment (seanfirefox) [inline docs/UserGuide.md:1]: Suggestion to standardise all `...` to `…​` instead.
image
image

**Review by seanfirefox — APPROVED** [review]
LGTM!

File: docs/UserGuide.md
Comment (mynameizzhafeez) [inline docs/UserGuide.md:1]: Screenshot 2023-04-03 at 4 01 54 PM

Under meet commands:
Final → finals.

Comment (mynameizzhafeez) [inline docs/UserGuide.md:1]: Screenshot 2023-04-03 at 4 02 47 PM

Not entirely accurate. It recommends multiple times and locations.

Comment (mynameizzhafeez) [inline docs/UserGuide.md:1]: Screenshot 2023-04-03 at 4 05 02 PM

Without affecting your real contacts.

**Review by mynameizzhafeez — CHANGES_REQUESTED** [review]
Make changes please :)

File: docs/UserGuide.md
[the remark this answers is not in the document]
Comment (russelltankaimin) [reply docs/UserGuide.md:1]: Rejected. Check this out from [thread](https://english.stackexchange.com/questions/137664/recommendation-of-vs-recommendation-for-what-is-the-difference) from English stack exchange.

[the remark this answers is not in the document]
Comment (zichen-3974) [reply docs/UserGuide.md:1]: ... view command retrieves and displays ....

[the remark this answers is not in the document]
Comment (russelltankaimin) [reply docs/UserGuide.md:1]: ok

**Review by zichen-3974 — APPROVED** [review]
LGTM.

**Review by mynameizzhafeez — APPROVED** [review]
LGTM. Well done Russell!

**Pull request merged**